### ISSUE TITLE
Switch from Mapbox to Leaflet

### ISSUE THREAD

**Issue by CoderTobi** [photoview/photoview#746]
**Is your feature request related to a problem? Please describe.**
Mapbox requires a Credit Card to get a Token. Not all people do have a Credit Card or want to get away the information.

**Describe the solution you'd like**
Other galleries like PiGallery2 let you switch between Leaflet and Mapbox. Leaflet does not require a Credit Card and in the Case of PiGallery2 not even the admin to enter a token.

**Describe alternatives you've considered**
It would be really great if you would be able to use Leaflet or any other map-provider that doesn't require a Credit Card. I obviously do not know how much effort it would take to change it.

**Comment by viktorstrate** [comment]
You can generate free Mapbox tokens without credit card for up to 50,000 monthly requests, which should be enough for most personal use cases.

I am reluctant to also add Leaflet as it would add a lot of duplicated funcitonality which would be harder to maintain.

**Comment by JPPalus** [comment]
No, you can't. You have to have a credit card to make a "free" account that allows you to generate a "free" token.
That need for an external token and the privacy issue it causes that I switched to another photo gallery...

**Comment by cleoold** [comment]
Mapbox actually requires a card. Just tried today. Do we have new plans to support the feature OP has asked for?

**Comment by kkovaletp** [comment]
There is the #879 issue, also asking for an alternative to the MapBox. I'd close this one and move the discussion there just to keep things consolidated.
